### PR TITLE
XEP-0202: Entity Time

### PR DESCRIPTION
**Pull request by ludwikbukowski**
mod_time

### CODE REVIEW

**Comment by ppikula** [comment]
Can you add some docs about it?: 
1. new file in  `doc/modules` with module description + https://github.com/esl/MongooseIM/blob/master/mkdocs.yml 
2. add XEP number to README and to https://github.com/esl/MongooseIM/blob/master/doc/user-guide/Features-and-supported-standards.md

**Comment by michalwski** [comment]
Also please try mkdocs locally before you push the new doc page. I usually use following command in MongooseIM's root directory:
```mkdocs serve```

**Comment by mongoose-im** [comment]
travis is using test branch `mod_time` from https://github.com/esl/ejabberd_tests/tree/mod_time

File: apps/ejabberd/src/mod_time.erl [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    IQ#iq{type = error, sub_el = [SubEl, ?ERR_NOT_ALLOWED]};
+
+process_local_iq(_From, _To, #iq{type = get} = IQ) ->
+    Now = now(),
+    Now_universal = calendar:now_to_universal_time(Now),
+    Now_local = calendar:now_to_local_time(Now),
+    {UTC_time, UTC_diff} = jlib:timestamp_to_iso(Now_universal, utc),
+    UTC = list_to_binary(UTC_time ++ UTC_diff),
+    Seconds_diff =
+    calendar:datetime_to_gregorian_seconds(Now_local) -
```
Comment (michalwski) [inline apps/ejabberd/src/mod_time.erl]: I'd export this calculation to dedicated function just for readability.

File: apps/ejabberd/src/mod_time.erl [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    calendar:datetime_to_gregorian_seconds(Now_universal),
+    {Hd, Md, _} =
+    calendar:seconds_to_time(abs(Seconds_diff)),
```
Comment (michalwski) [inline apps/ejabberd/src/mod_time.erl]: Let's merge lines 40 and 41 to one.

File: apps/ejabberd/src/mod_time.erl (hunk 1 above)
Comment (michalwski) [inline apps/ejabberd/src/mod_time.erl]: Actually I meant this:
```erlang
Seconds_diff =
calendar:datetime_to_gregorian_seconds(Now_local) -
calendar:datetime_to_gregorian_seconds(Now_universal),
```

What you did is a good change but I still what to see above 3 lines done more readably  :)

**Pull request merged**